spec/requests/distributions_requests_spec.rb

```diff
@@ -143,7 +143,7 @@
       end
 
       it "sums distribution totals accurately" do
-        distribution = create(:distribution, :with_items, item_quantity: 0)
+        distribution = create(:distribution, :with_items, item_quantity: 1)
```

Comment:
Made this change because you quantity needs to be positive when you create a distribution.

Reply:
I looked into this a bit, I was worried for a minute that "0" zero might be a legit distribution -- they requested 17 of something and the bank decided to give them 0 (but maybe give them some other items they requested). But @cielf pointed out that they can _remove_ those items from the distribution -- so I think we're fine.